### PR TITLE
update API code to clean up WF and TET tags after deleting a main PDF file

### PR DESCRIPTION
**Pull request by sweng66**

### CODE REVIEW

File: agr_literature_service/api/crud/referencefile_crud.py [hunk 1]
```diff
@@ -76,7 +77,7 @@ def get_main_pdf_referencefile_ids_for_ref_curies_list(db: Session, curies: List
     curie_main_ref_file_map = {}
 
     for ref_file in all_ref_files:
-        if ref_file.file_class == "main" and ref_file.file_publication_status == "final" and ref_file.pdf_type == "pdf":
+        if ref_file.file_class == "main" and ref_file.file_publication_status == "final" and ref_file.file_extension == "pdf":
```
Comment (valearna) [inline agr_literature_service/api/crud/referencefile_crud.py:80]: same here

**Comment by sweng66** [comment]
@valearna Good catch! I didn't realize that WB has different pdf types. 
The reason I did this fix is due to the NULL pdf_type for some main PDFs (eg some ZFIN pdfs).

Can we change code to check if pdf_type is None or pdf_type == 'pdf'?

File: agr_literature_service/api/crud/referencefile_crud.py [hunk 2]
```diff
@@ -49,7 +50,7 @@ def get_main_pdf_referencefile_id(db: Session, curie_or_reference_id: str,
                                               load_referencefiles=True)
     main_pdf_referencefiles = [referencefile for referencefile in reference.referencefiles if
                                referencefile.file_class == "main" and referencefile.file_publication_status == "final"
-                               and referencefile.pdf_type == "pdf" and referencefile.file_extension == "pdf"]
+                               and referencefile.file_extension == "pdf"]
```
[the remark this answers is not in the document]
Comment (sweng66) [reply agr_literature_service/api/crud/referencefile_crud.py]: Good catch! Fixed

File: agr_literature_service/api/crud/referencefile_crud.py (hunk 1 above)
Comment (sweng66) [reply agr_literature_service/api/crud/referencefile_crud.py]: Fixed. =)

**Comment by sweng66** [comment]
@valearna fixed the code to check the file_extension instead of the pdf_type as discussed. thanks.

**Review by valearna — APPROVED** [review]
Everything looks good to me now, thanks @sweng66 !

**Comment by sweng66** [comment]
Thanks @valearna

**Pull request merged**